proposal/more-math-functions.md

```diff
@@ -25,6 +27,54 @@ This proposal adds the following members to the built-in `sass:math` module.
     * [`atan2()`](#atan2)
       * [Edge cases](#edge-cases)
 
+## Background
+
+> This section is non-normative.
+
+Sass recently implemented a module system with a new built-in `sass:math`
+module. The demand for built-in math functions can now be fulfilled safely by
+implementing them inside this module. None of these new functions will be made
+available on the global namespace.
+
+## Summary
+
+> This section is non-normative.
+
+This proposal defines Sassified versions of all the mathematical functions in
+the [CSS Values and Units 4 Draft][], as well as logarithms and the constants
+`e` and `pi`. Each function is basically equivalent to its mathematical form,
+with stricter unit handling. Proper unit handling prevents these functions from
+creating meaningless units. For instance, what is the unit of `(1px)`<sup><code>2</code></sup>? A `px`<sup><code>2</code></sup>?
```

Comment:
For the example, it might be better to do something with fractional exponents, since `px^2` is actually a reasonable unit of area (even if it's not really useful for CSS). It might also make more sense to just use a `^` to represent the exponent here since GitHub doesn't support proper math rendering.

Also, make sure to break this line at 80 characters

Reply:
Good point. Changed this to `px^(1/3)`.